puppet/zulip_ops/manifests/apache.pp

```diff
@@ -18,7 +18,7 @@
     mode    => '0600',
   }
 
-  file { '/etc/apache2/users/wiki':
+  file { '/etc/apache2/users/monitoring':
```

Comment:
I think we need to update `puppet/zulip_ops/templates/nagios_apache_site.conf.template.erb` as well

(From `git grep wiki puppet/`)

Otherwise lgtm.

Reply:
Whoops -- thanks for the catch.